flask_dance/contrib/google.py

```diff
@@ -45,6 +46,9 @@ def make_google_blueprint(
         reprompt_consent (bool): If True, force Google to re-prompt the user
             for their consent, even if the user has already given their
             consent. Defaults to False
+        reprompt_select_account (bool): If True, force Google to re-prompt the select account page,
+         even if there is a single logged-in user. Defaults to False
+            consent. Defaults to False
```

Comment:
Too much copy paste?

Reply:
I tried to keep the same syntax. Can reformulate if you want?

Reply:
Good catch.